features/api_full_token_attach.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Attach API endpoint
```

Comment:
Could we have a test here that attaches the machine and uses the option to not enable the services by default, just to assure the functionality on the API?